tests/scripts/run_profiler_regressions.sh

```diff
@@ -19,41 +19,30 @@ run_profiling_test(){
 
     remove_default_log_locations
 
-    $PROFILER_SCRIPTS_ROOT/profile_this.py -c "pytest -svvv $TT_METAL_HOME/tests/tt_eager/python_api_testing/sweep_tests/pytests/tt_dnn/test_composite.py::test_run_eltwise_composite_test[lerp_binary-input_shapes0]"
+    $PROFILER_SCRIPTS_ROOT/profile_this.py -c "pytest tests/tt_eager/python_api_testing/sweep_tests/pytests/tt_dnn/test_matmul.py::test_run_matmul_test[BFLOAT16-input_shapes0]"
 
-    runDate=$(ls $PROFILER_OUTPUT_DIR/ops/)
+    runDate=$(ls $PROFILER_OUTPUT_DIR/)
 
-    ls $PROFILER_OUTPUT_DIR/ops/$runDate/ops_perf_results_$runDate.csv
-    ls $PROFILER_OUTPUT_DIR/ops/$runDate/ops_perf_results_$runDate.tgz
+    res=$(cat $PROFILER_OUTPUT_DIR/$runDate/ops_perf_results_$runDate.csv |awk -F, 'NR > 1 { print $22 }')
```

Comment:
I would personally put this into a smaller function that can have the script exit with an error code rather than `$()` - I noticed that failing commands will still allow scripts to continue with `$()`. Is that what you want?

Reply:
The arithmetic check on res will catch other output ons res. 

The code wasn't getting repeated so transferring all the necessary args for one time use didn't seem necessary.